### DISCUSSION TITLE
How to change resolution/framerate  for all connected clients ?

### DISCUSSION THREAD

**Discussion by hugoperier** [aiortc/aiortc#843] [Q&A]
Hi, from the webcam example, I would like to change the resolution or the framerate of the MediaPlayer without the need of disconnecting and reconnecting clients.
As I understood, there is no way to update ffmpeg parameter at runtime, I tried to stop the MediaPlayer to create it again with updated settings.
I tried the following code without any success:

```
webcam.video.stop()
webcam = MediaPlayer(
                    "default:none", format="avfoundation", options=options
                )
relay = MediaRelay()
video = relay.subscribe(webcam.video)
for pc in pcs:
            pc.addTrack(video)
```

I get this error:
`ioctl(VIDIOC_QBUF): Invalid argument`

I think I am going with the wrong approach, could you please give me a hint on how to do this ?

**Comment by LeehanLee** [comment]
anyone has any idea? Please
